src/components/UI/Header/Header.styled.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const HeaderStyled = styled.header`
+	background-color: black;
+	color: white;
+	height: 5rem;
+	border-bottom: solid white;
+	width: 100%;
```

Comment:
I guess you don't need this width here, since headers have by default this property.

Reply:
Fixed